realm/realm-library/src/objectServer/java/io/realm/internal/objectserver/SyncSession.java

```diff
@@ -87,7 +86,6 @@
  *
  * This object is thread safe.
  */
-@Keep
```

Comment:
I introduced `KeepMember` annotation that only keeps single method or field, and then marked `notifySessionError` with it instead of adding `Keep` to the class since `Keep` prohibits to rename all classes used in method signature in the class.